docs/file-formats/excel2json.md

```diff
@@ -77,6 +80,34 @@ Then, use the following command:
 dsp-tools new-excel2json data_model_files project.json
 ```
 
+## The `json_header.xlsx`
+
+The header section can be filled out with the "json_header.xlsx" file.
+The file must be in the main folder, and the name must be an exact match.
+
+It is recommended to use the [template](../assets/data_model_templates/json_header.xlsx).
+
+The sheets should be filled as follows:
+
+- **Prefixes**
+    - Both columns: prefixes and URI are mandatory.
+    - At least one prefix has to be declared
```

Comment:
No I made prefixes mandatory, I discussed this with Daniela, because we want to strongly encourage references to external ontologies, we can remind people by making prefixes mandatory. But I would not go so far as to make them mandatory in the json, that would be a breaking change, here it is not.